src/modules/sdk/objections/service.ts

```diff
@@ -106,3 +106,16 @@ export const getAttachment = async (companyNumber: string,
   const response = await makeAPICall(axiosConfig);
   return response.data as Attachment;
 };
+
+export const deleteAttachment = async (companyNumber: string,
+                                       token: string,
+                                       objectionId: string,
+                                       attachmentId: string): Promise<void> => {
```

Comment:
you should be able to remove the Promise<void> return type. By marking it async it should auto wrap the return type in a promise and if left blank should return void anyway.